files/galaxy/tpv/tools.yml

```diff
@@ -303,6 +303,8 @@ tools:
       require:
         - docker
 
+  toolshed.g2.bx.psu.edu/repos/iuc/chewbbaca_createschema/.*:
```

Comment:
what about the other chewbbaca tools?

Reply:
If I add `toolshed.g2.bx.psu.edu/repos/iuc/chewbbaca*` does that work for all chewbbaca packages?

Reply:
I think so. @sanjaysrikakulam ?

Reply:
We can give it a shot. To my understanding this `toolshed.g2.bx.psu.edu/repos/iuc/chewbbaca.*` should work. I have made a suggestion to add a `.` to the regex.

Reply:
Thanks @sanjaysrikakulam